### DISCUSSION TITLE
Deleted My Account.

### DISCUSSION THREAD

**Discussion by anandpanda** [community/community#72258] [GitHub Education]
### Hi everyone,

Due to someone, I deleted the account with which I have claimed student's pack.
And now, after creating a fresh new github account, I was trying to apply for the students pack, but it is rejecting, saying "Your uploaded proof has already been used previously".
What do I do.
Please help, please.

**Comment by omarchouman** [comment]
### Hello, Potential Steps You Could Take:

### Contact GitHub Support:

- Go to the GitHub Support page [https://support.github.com/contact](https://support.github.com/contact).
- Choose the "Education and verification" category.
- Describe your issue, explaining that you deleted your previous account and are now unable to apply for the student pack with your new account because the proof has already been used.

### Provide Documentation:

- If you have any documentation related to your student status, such as a student ID, enrollment letter, or any other proof, be prepared to provide it.
- Clearly explain the situation and attach any relevant documents to support your case.
